### PR TITLE
DEVDOCS-52: URL Forwarding

### PR DESCRIPTION
**Pull request by brenaandring**
Resolves: [DEVDOCS-52](https://fastaf.atlassian.net/browse/DEVDOCS-52)

# What does this pr do?
This pull request adds redirect from links of old docs to redirect to new docs

Old urls correspond to new urls: [URL Forwarding Spreadsheet](https://docs.google.com/spreadsheets/d/1KJHw7d8LSrbf1AsIDXwfOVxruqLCEs0qgBZdPTSewgs/edit?pli=1#gid=0)

### CODE REVIEW

File: developer-portal/for-developers/custom-integration/fast-api/overview.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 title: fast.js Docs
 description: Getting Started with Fast with fast.js
 keywords: fast.js
-
 enableToc: true
 redirectFrom:
   - /developer-portal/fast-api/
+  - /order/order/
+  - /merchant/merchant/
+  - src/pages/order/order/
```
Comment (adamaltman) [inline developer-portal/for-developers/custom-integration/fast-api/overview.md:10]: Should start with a leading slash.

File: developer-portal/for-developers/bigcommerce/adding-fast-button-to-blog-posts/adding-code.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 title: "Fast BigCommerce Docs: Adding Fast Buttons to Blog Posts"
 description: Adding Fast Buttons to BigCommerce Blog Posts.
 keywords: Adding Fast Buttons to BigCommerce Blog Posts
+
+enableToc: true
+redirectFrom:
+  - /developer-portal/bigcommere-checkout-blog-posts/
```
Comment (adamaltman) [inline developer-portal/for-developers/bigcommerce/adding-fast-button-to-blog-posts/adding-code.md:8]:
```suggestion
  - /developer-portal/bigcommerce-checkout-blog-posts/
```
Did the original have a typo or is this a typo?

File: developer-portal/for-developers/woocommerce/pre-install/gather-your-docs.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 title: "Fast WooCommerce Docs: Gather Your Docs"
 description: Getting Started with Fast on WooCommerce! Quickstart
 keywords: Fast WooCommerce Quickstart
+
+enableToc: true
+redirectFrom:
+  - /developer-portal/sign-up-with-woocommerce/
+  - /developer-portal/woocommerce-install-checkout/#capture-your-app-id
```
Comment (adamaltman) [inline developer-portal/for-developers/woocommerce/pre-install/gather-your-docs.md:9]: You can redirect pages, not anchor links within pages.

**Comment by kaitlynhova** [comment]
Thank you for the helpful review, @adamaltman !!! 🚀

**Comment by adamaltman** [comment]
The redirect works for me. Here is an example:

https://preview.redoc.ly/fastaf/DEVDOCS-52_brena_url-forwarding/developer-portal/bigcommerce/

Redirects to:

https://preview.redoc.ly/fastaf/DEVDOCS-52_brena_url-forwarding/developer-portal/for-developers/bigcommerce/

**Pull request merged**